ms2/src/org/labkey/ms2/protein/tools/GoLoader.java

```diff
@@ -69,24 +66,25 @@ public abstract class GoLoader
     private static Boolean _goLoaded = null;
     private static GoLoader _currentLoader = null;
 
-    private final StringBuffer _status = new StringBuffer();  // Can't use StringBuilder -- needs to be synchronized
+    private final HtmlStringBuilder _status = HtmlStringBuilder.of();  // Can't use StringBuilder -- needs to be synchronized

```

Comment:
Comment suggests we need to synchronize access to `_status`. Should either add this or remove the comment.

Reply:
Hmmm

Reply:
Turns out GO loading is quite broken: https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=49301

Send fine to just merge as is.